core/domain/collection_domain.py

```diff
@@ -318,6 +398,18 @@ def create_default_collection(
     def from_dict(
             cls, collection_dict, collection_version=0,
             collection_created_on=None, collection_last_updated=None):
+        """Return a Collection domain object from a dict.
```

Comment:
"Returns" for parity with other docstrings in this file. Ditto elsewhere; please check.